cmd/aries-js-worker/package-lock.json

```diff
@@ -0,0 +1,4457 @@
+{
+  "name": "@trustbloc/aries",
```

Comment:
Do we need package-lock.json file?

Reply:
I think the answer would be yes: consistency in the development workflow and consistency in the CI.

Reply:
@llorllale where did we land on this topic?

Reply:
Yes, reproducible builds are good. For instance, notice that not all of the transitive dependency versions are locked down.